README.md

```diff
@@ -76,64 +190,111 @@ There are now more testing options! You can test the client, or the server or bo
 
 From the `client` directory:
 
-* `ng test` runs the client tests.
-* `ng test --code-coverage` runs the client tests and generates a coverage file you can find in your client directory `client/coverage/client/index.html`.
-Right click on `index.html` and select `Copy path` and paste it into your browser of choice. For Chrome users, you can drag and drop `index.html` onto the tab area of Chrome and it will open it.
+- `ng test` runs the client tests
+  - This will pop up a Chrome window with the results of the tests.
+  - This will run "forever", updating both in your terminal and in the Chrome
+    window that gets generated. Typing CTRL-C in the terminal window will end
+    the `ng test` process and close the generated Chrome window.
+  - You can add `ng test --watch=false` if you just want to run the tests once
+    instead of going into the "run forever" mode.
+- `ng test --code-coverage` runs the client tests and generates a coverage report
+  - It generates a coverage report you can find in your client directory `client/coverage/client/index.html`.
+  - Right click on `index.html` and select `Copy path` and paste it into your browser of choice. You can also drag and drop `index.html` onto the tab area of your browser and it will open it.
+
+#### Linting the client
+
+We have included a tool called ESLint which helps analyze the client
+TypeScript and JavaScript code and catch various errors and concerns. You will most likely see it directly in VS Code as yellow and red underlines. You can also directly run the linter on the entire client by running `ng lint` in the terminal in the `client` directory. This will check the whole client project and tell you if there are any issues.
```

Comment:
```suggestion
We have included a tool called ESLint which helps analyze the client
TypeScript and template HTML code and catch various errors and concerns. You will most likely see it directly in VS Code as yellow and red underlines. You can also directly run the linter on the entire client by running `ng lint` in the terminal in the `client` directory. This will check the whole client project and tell you if there are any issues.
```